fastlane/download_metadata.swift

```diff
@@ -57,6 +57,22 @@ let languages = [
     "zh-Hant": "zh-tw",
 ]
 
+func shouldUpdateSubtitle(locale: String) -> Bool {
+    return isIn20211210Experiment(locale: locale) == false
+}
+
+func shouldUpdateKeywords(locale: String) -> Bool {
+    return isIn20211210Experiment(locale: locale) == false
+}
+
+func isIn20211210Experiment(locale: String) -> Bool {
+    // The experiment should run on every English locale.
```

Comment:
Worth adding the shortened link to the P2 post here?

Reply:
Maybe the obfiscated ref for it at least (i.e. `pdnsEh-5Q-p2`)?

Reply:
```suggestion
    // The experiment should run on every English locale. See details in P2 post ref `pdnsEh-5Q-p2`.
```